javascript/ql/test/experimental/PoI/CommandInjectionPoIConfiguration.ql

```diff
@@ -0,0 +1,15 @@
+/**
+ * @kind problem
+ */
+
+import javascript
+import experimental.PoI.PoI::PoI
+import semmle.javascript.security.dataflow.CommandInjection
+import semmle.javascript.security.dataflow.IndirectCommandInjection
+import semmle.javascript.security.dataflow.ShellCommandInjectionFromEnvironment
+
+class Config extends StandardPoIConfigurations::DataFlowConfigurationPoIConfiguration {
```

Comment:
Again, `StandardPoIConfigurations::DataFlowConfigurationPoIConfiguration` is quite a mouthful.

Reply:
I have kept the two `StandardPoI...` modules, but their content is now exposed when `PoI` is imported. So you just have to type `extends DataFlowConfigurationPoIConfiguration` now. 
Or do you also think that the `DataFlowConfigurationPoIConfiguration` is too long?